docs/content/quickstart/oracle-tutorial.md

```diff
@@ -0,0 +1,143 @@
+# Demo: Oracle CDC to Elasticsearch
+
+**1. Create `docker-compose.yml` file using following contents:**
+
+```
+version: '2.1'
+services:
+  oracle:
+    image: yuxialuo/oracle-xe-11g-r2-cdc-demo:v1.0
+    ports:
+      - "1521:1521"
+  elasticsearch:
+    image: elastic/elasticsearch:7.6.0
+    environment:
+      - cluster.name=docker-cluster
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - discovery.type=single-node
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+  kibana:
+    image: elastic/kibana:7.6.0
+    ports:
+      - "5601:5601"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+```
+The Docker Compose environment consists of the following containers:
+- Oracle: Oracle 11g and a pre-populated `products` and `orders` table in the database.
+- Elasticsearch: store the join result of the `orders` and `products` table.
+- Kibana: mainly used to visualize the data in Elasticsearch
+
+To start all containers, run the following command in the directory that contains the docker-compose.yml file.
+```shell
+docker-compose up -d
+```
+This command automatically starts all the containers defined in the Docker Compose configuration in a detached mode. 
+Run docker ps to check whether these containers are running properly. You can also visit http://localhost:5601/ to see if Kibana is running normally.
+
+Don’t forget to run the following command to stop all containers after you finished the tutorial:
+```shell
+docker-compose down
+```
+
+**2. Download following JAR package to `<FLINK_HOME>/lib`**
+
+*Download links are available only for stable releases.*
+
+- [flink-sql-connector-elasticsearch7_2.11-1.13.2.jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-elasticsearch7_2.11/1.13.2/flink-sql-connector-elasticsearch7_2.11-1.13.2.jar)
+- [flink-sql-connector-oracle-cdc-2.2-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-oracle-cdc/2.2-SNAPSHOT/flink-sql-connector-oracle-cdc-2.2-SNAPSHOT.jar)
+
+**3. Launch a Flink cluster and start a Flink SQL CLI**
+
+Execute following SQL statements in the Flink SQL CLI:
+
+```sql
+-- Flink SQL
+-- checkpoint every 3000 milliseconds                       
+Flink SQL> SET execution.checkpointing.interval = 3s;
+
+Flink SQL> CREATE TABLE products (
+    ID INT,
+    NAME STRING,
+    DESCRIPTION STRING,
+    PRIMARY KEY (ID) NOT ENFORCED
+  ) WITH (
+    'connector' = 'oracle-cdc',
+    'hostname' = 'localhost',
+    'port' = '1521',
+    'username' = 'flinkuser',
+    'password' = 'flinkpw',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)